package.json

```diff
@@ -57,10 +57,10 @@
     "clean": "rimraf dist",
     "compile": "tsc",
     "completion": "tabtab install --name jhipster --auto",
-    "copy-files": "cpy 'generators/**' '!**/*.(js|cjs|mjs|ts|cts|mts|snap)' '!**/__*/*' dist/generators && cpy 'package.json' dist",
+    "copy-files": "cpy 'generators/**' '!**/*.(js|cjs|mjs|ts|cts|mts|snap)' '!**/__*/*' dist/generators",
```

Comment:
Only the folder `dist` is set in `files`. Will this not cause issues when including JHipster because the package.json is not part of the includable files?

Reply:
No, some files are always included, https://docs.npmjs.com/cli/v8/configuring-npm/package-json#files

Reply:
Sounds good.